docs/html/cli/pip_install.rst

```diff
@@ -1225,3 +1215,6 @@ Examples
 
 .. [1] This is true with the exception that pip v7.0 and v7.0.1 required quotes
        around specifiers containing environment markers in requirement files.
+
+
+.. _PEP_518: https://github.com/django-money/django-money/issues/595
```

Comment:
Some kind of copy-paste error here - should have been https://www.python.org/dev/peps/pep-0518/

Reply:
Duly noted! The original reference can be updated to be:

```
:pep:`518`
```

Which is much easier to maintain, and avoid errors like this. :)